userguide/network.rst

```diff
@@ -332,6 +332,7 @@ summarizes the options available when configuring IPMI with the
    |                      |                |                                                                              |
    +----------------------+----------------+------------------------------------------------------------------------------+
    | Password             | string         | Enter the password used to connect to the IPMI interface from a web browser. |
+   |                      |                | Has a maximum length of 20 characters.                                       |
```

Comment:
The second part should be a complete sentence:
"The maximum length is 20 characters." or "Password length is limited to 20 characters."